doc/guides/device_mgmt/mcumgr.rst

```diff
@@ -19,6 +19,7 @@ over the following transports:
 
 * BLE (Bluetooth Low Energy)
 * Serial (UART)
+* UDP (IPv4 and IPv6)
```

Comment:
I am not sure whether IP version should be added here; supported IP version depends on stack rather than mcumgr. Yes, mcumgr supports UDP, but IP depends on system.

Reply:
I added this, because this is explicitly implemented in zephyr. Should i just change it to UDP and remove the (IPv4 and IPv6)?

Reply:
Maybe just change it to UDP over IP ?